### PR TITLE
UX Move card not only by the header

### PR DESCRIPTION
**Pull request by KenAlin**

### CODE REVIEW

File: src/components/cardPreview/CardPreview.jsx [hunk 1]
```diff
@@ -180,7 +179,8 @@ class Card extends Component {
 
   render() {
     return (
-      <UICard title={this.getHeader()} extra={this.getDropdown()} className="card">
+      <UICard title={this.getHeader()} extra={this.getDropdown()} 
+      {...this.props.dragHandleProps}  className="card">
```
Comment (TheoGauchoux) [inline src/components/cardPreview/CardPreview.jsx]: Need a better indentation

File: src/components/cardPreview/CardPreview.jsx [hunk 2]
```diff
@@ -17,7 +17,6 @@ class Card extends Component {
         <EditField
           text={this.props.title}
           save={(newTitle) => { this.props.saveCardTitle(this.props.id, newTitle) }}
-          dragHandleProps={this.props.dragHandleProps}
```
Comment (TheoGauchoux) [inline src/components/cardPreview/CardPreview.jsx:20]: You remove this prop, but in `EditField` it's always required.

**Review by TheoGauchoux — DISMISSED** [review]

File: src/components/cardPreview/CardPreview.jsx (hunk 2 above)
Comment (KenAlin) [reply src/components/cardPreview/CardPreview.jsx:20]: That's useless : you can still drag&drop cards by the header without this props.

File: src/components/cardPreview/CardPreview.jsx (hunk 1 above)
Comment (KenAlin) [reply src/components/cardPreview/CardPreview.jsx]: done

File: src/components/cardPreview/CardPreview.jsx (hunk 2 above)
Comment (KenAlin) [reply src/components/cardPreview/CardPreview.jsx:20]: Nevermind ! I noticed the error in JS console. Fixed in commit e8dd1c8

**Review by Tombean — APPROVED** [review]

**Pull request merged**